src/code/z_camera.c

```diff
@@ -8054,7 +8062,12 @@ s16 func_8005AD1C(Camera* camera, s16 arg1) {
     return camera->unk_14C;
 }
 
-s32 Camera_ChangeDoorCam(Camera* camera, Actor* doorActor, s16 camDataIdx, f32 arg3, s16 timer1, s16 timer2,
+/**
+ * A bgCamDataIndex of -99 will save the door params without changing the camera setting
+ * A bgCamDataIndex of -1 uses the default door camera setting (CAM_SET_DOORC)
```

Comment:
I see that you mentioned -1 in the transition actor struct but not -99, is there a particular reason for this or it's just not relevant in that context?

Reply:
It seemed less relevant based on the context, and the comment line in the transition actor struct was already getting really long. I think it would be more appropriate to include the -99, or maybe I could just remove all 3 cases and leave it to looking up the function?